README.md

```diff
@@ -36,6 +36,10 @@ The following types of data are being considered:
  | URL         | Url with more information on the image|
  | Summary     | Short Description of the image|
  | Description | Detailed description of the image|
+ | VcsType    | Where the image source can be obtained from a version control system, which software is used|
```

Comment:
The important piece is that we have to also standardize the things that go into the label in this case otherwise it isn't super useful.